src/proto/console/text/output.rs

```diff
@@ -1,3 +1,4 @@
+use core::fmt;
 use crate::{Result, Status};
 
 /// Interface for text-based output devices.
```

Comment:
We should add a small note here, to point out to people that they can use the standard `write`/`writeln` macros to print stuff.

Reply:
Done :)